### PR TITLE
PT-164102948 Remove legacy WS API

### PR DESCRIPTION
**Pull request by cytadela8**
Related node PR aeternity/aeternity#2266

### CODE REVIEW

File: node/api/README.md [hunk 1]
```diff
@@ -64,14 +67,171 @@ connections will be queued. The queue has a maximum size and when it is
...
+Implementation support exists for batch processing, however this is not yet tested.
+
+### Examples
+
+#### Example, synchronous RPC
+
+```javascript
+{
+  "jsonrpc": "2.0",
+  "id": -576460752303423475,
```
Comment (lucafavatella) [inline node/api/README.md]: Is the negative id expected?

Comment (lucafavatella) [reply node/api/README.md]: I see a number https://www.jsonrpc.org/specification#request_object

so negative looks alright...

File: node/api/README.md [hunk 2]
```diff
@@ -64,14 +67,171 @@ connections will be queued. The queue has a maximum size and when it is
...
+        "contract": "ct_28pwhjFEVfuirjPzJDRTE4o4ybjKw7vjMymRdhhVrDEVTuXAtq",
+        "round": 8
+      }
+    }
+  },
+  "id": null,
+  "jsonrpc": "2.0"
+}
+```
+Detailed message transcripts from test suites can be found [for JSON-RPC](./examples/aehttp_integration_SUITE/json-rpc/) and [for legacy](./examples/aehttp_integration_SUITE/legacy/).
```
Comment (lucafavatella) [inline node/api/README.md]:
```suggestion
Detailed message transcripts from test suites can be found [for JSON-RPC](./examples/aehttp_integration_SUITE/json-rpc/).
```

**Comment by cytadela8** [comment]
Blocked till #349 is merged

**Comment by cytadela8** [comment]
Rebased, most of removing legacy documentation was done in #349. This change finishes the task.

File: node/api/README.md [hunk 3]
```diff
@@ -64,14 +67,171 @@ connections will be queued. The queue has a maximum size and when it is
...
+        "contract": "ct_28pwhjFEVfuirjPzJDRTE4o4ybjKw7vjMymRdhhVrDEVTuXAtq",
+        "round": 8
+      }
+    }
+  },
+  "id": null,
+  "jsonrpc": "2.0"
+}
+```
+Detailed message transcripts from test suites can be found [here](./examples/aehttp_integration_SUITE/json-rpc/).
```
Comment (lucafavatella) [inline node/api/README.md]: The link seems not working for me, maybe you meant the following?
```suggestion
Detailed message transcripts from test suites can be found [here](./examples/channels/json-rpc/).
```

File: node/api/channels_api_usage.md [hunk 4]
```diff
@@ -62,12 +62,11 @@ In order to use a channel, it must be opened. Both parties negotiate parameters
...
 
-The channel websocket api currently supports two protocols: [`json-rpc`](https://www.jsonrpc.org/specification) and `legacy`. Which protocol to use can be specified with the `protocol` option.
- If omitted, `legacy` is chosen by default, although this default is likely to change in the future.
+The channel websocket api currently supports one protocol: [`json-rpc`](https://www.jsonrpc.org/specification). `legacy` protocol was removed. Choosen protocol has to be specified with the `protocol` option.
 
- In the examples below, the `legacy` protocol is used. For a description on how to translate into the `json-rpc` format, see [channel_ws_api_json-rpc.md](channel_ws_api_json-rpc.md).
+In the examples below, the `json-rpc` protocol is used.
 
-Detailed message transcripts from test suites can also be found [for JSON-RPC](./examples/aehttp_integration_SUITE/json-rpc/) and [for `legacy`](./examples/aehttp_integration_SUITE/legacy/)
+Detailed message transcripts from test suites can also be found [here](./examples/aehttp_integration_SUITE/json-rpc/).
```
Comment (lucafavatella) [inline node/api/channels_api_usage.md]:
```suggestion
Detailed message transcripts from test suites can also be found [here](./examples/channels/json-rpc/).
```
?

**Comment by lucafavatella** [comment]
Looks fine for the rest.

File: node/api/README.md (hunk 3 above)
Comment (cytadela8) [reply node/api/README.md]: Yes, sorry.

**Pull request merged**